File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/startCollectingTrieStats.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x0000000000000000000000000000000000000000"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.start_collecting_trie_stats(address)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/startCollectingTrieStats.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const address = "0x0000000000000000000000000000000000000000";

  provider.debug
    .startCollectingTrieStats(address, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
